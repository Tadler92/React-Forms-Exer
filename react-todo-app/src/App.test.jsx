import {render, fireEvent, waitFor} from "@testing-library/react";
import App from "./App";
import { describe, expect, it } from "vitest";

describe("Todo App Component", () => {
  it("renders without crashing", function () {
    render(<App />);
  });
  
  
  it("matches snapshot", function () {
    const {asFragment} = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});