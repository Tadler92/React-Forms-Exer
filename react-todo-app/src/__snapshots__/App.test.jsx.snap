// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Todo App Component > matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <div>
      <h3>
        Todo App
      </h3>
      <form
        class="NewTodoForm"
      >
        <label
          for="taskTodo"
        >
          Task Todo: 
        </label>
        <input
          id="taskTodo"
          name="taskTodo"
          placeholder="Task"
          type="text"
          value=""
        />
        <button>
          Add Todo
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
